en/Alert.md

```diff
@@ -0,0 +1,22 @@
+### Alert
+
+General guidelines for alerts and trips:
+* If a TripDescriptor is specified in an alert, the informed_entities must provide a `trip_id`. 
+* When a TripDescriptor doesn’t specify `start_date` or `start_time`, all trips with the corresponding `trip_id` are affected.

```

Comment:
This seems to belong in the spec as it is more about functionality and less about best practice.

Reply:
Addressed in https://github.com/MobilityData/GTFS_Realtime_Best-Practices/commit/26dd2eb173316bcf3caaaf93a9a0c9481103bc42.